src/comment.rs

```diff
@@ -58,25 +58,23 @@ fn custom_opener(s: &str) -> &str {
 }
 
 impl<'a> CommentStyle<'a> {
-    /// Returns `true` if the commenting style covers a line only.
+    /// Returns `true` if the commenting style is a single line comment.
     pub(crate) fn is_line_comment(&self) -> bool {
-        match *self {
+        matches!(
+            self,
             CommentStyle::DoubleSlash
-            | CommentStyle::TripleSlash
-            | CommentStyle::Doc
-            | CommentStyle::Custom(_) => true,
-            _ => false,
-        }
+                | CommentStyle::TripleSlash
+                | CommentStyle::Doc
+                | CommentStyle::Custom(_)
+        )
```

Comment:
I don't have any concerns with this, though I will share for awareness that there are cases where we've intentionally not converted to `matches!` because of how rustfmt currently formats them (we don't have special casing so the args are treated as standard macro args and thus get formatted as bin expressions instead of a pattern. So in some cases, including this one, switching to the macro has some negative impacts on readability as it adds rightward drift

Reply:
Thanks for the tip, I agree that readability is never as simple as *always use X over Y*, even if X is generally regarded as the better thing. It's always case by case.

In this case, I think the removal of the false branch structure is worth the rightward drift.